data/default_okd_release_images.json

```diff
@@ -2,6 +2,7 @@
     {
         "openshift_version": "4.11",
         "cpu_architecture": "x86_64",
+        "cpu_architectures": ["x86_64"],
```

Comment:
@vrutkovs this change would benefit from your approval

Reply:
LGTM